tests/integration/conftest.py

```diff
@@ -240,6 +240,23 @@ def fail_on_errors():
         yield app
 
 
+def fail_missing_secrets(msg):
+    """Fail or skip the test because of missing secrets.
+
+    Secrets are only guaranteed to be available on CI when the branch being tested is
+    committed to the main getsentry repository.  Especially for forks they are hidden to
+    stop external contributors from stealing the secrets.
+
+    This will call `pytest.skip` of the secrets are missing but this is acceptable because
+    e.g. we're on a developer's machine or on a forked repository.  If running on CI in the
+    main repository this will call `pytest.skip`.
```

Comment:
```suggestion
    This will call `pytest.skip` if the secrets are missing but this is acceptable because
    e.g. we're on a developer's machine or on a forked repository.  If running on CI in the
    main repository this will call `pytest.fail`.
```